src/server/acl/user_registry.h

```diff
@@ -51,7 +51,8 @@ class UserRegistry {
 
   // Acquires a read lock
   // Used by Auth
-  bool AuthUser(std::string_view username, std::string_view password) const;
+  std::pair<bool, const std::string_view> AuthUser(std::string_view username,
+                                                   std::string_view password) const;
```

Comment:
Please add a comment what the return value means (not obvious without looking at impl how to tell apart non-existing and existing but disabled user)

Reply:
No needed anymore, we deep copy the username :)